### DISCUSSION TITLE
Using T5 model for NER

### DISCUSSION THREAD

**Discussion by baivabdash** [explosion/spaCy#10299] [Help: Coding & Implementations]
Hi, 

While using 't5-small' model from huggingface hub i am getting following error.

_Valueerror: You have to specify either decoder_input_ids or decoder_input_embeds._
![spacy_quesn](https://user-images.githubusercontent.com/77936888/154215123-cdeca456-245b-4dad-af32-28e22326f80a.PNG)

I am trying this to train a custom NER pipeline. Any pointers would be helpful.

Thanks,

**Comment by polm** [comment]
Please don't past screenshots of code, paste it as text.

What did you modify to use t5-small? Please share the config/code you used for that. 

If you haven't done so, I recommend trying to train an NER model using the default settings first to make sure nothing is wrong with your environment in general.

**Reply by polm** [reply]
We just added [an FAQ entry](https://github.com/explosion/spaCy/discussions/10327) about using different Transformers models that may help you. 

That said, I'm not sure that using t5 as base for NER makes sense, since the main selling point of the model is its NLG capabilities.

**Reply by Premshay** [reply]
I'm also facing the same error with another model.
I understand that I need to customize some code, but for the life of me I can't trace back to the proper location.
We need to change the input_ID.
If I share the current code situation will you be u be able to point in the right direction?

**Reply by polm** [reply]
Please don't reply to old threads - it's better to open a new thread, describe your problem, and link to an old thread if a related one exists. 

When opening a new thread, please include more information - what transformer model are you trying to use, what do you mean you "need to change the input_ID", what did you do that gave an error?
